### PR TITLE
Update page titles

### PR DESCRIPTION
**Pull request by weapdiv-david**
### Changes

Updates page titles as per EQ-230
### How to test

1) Check out the branch
2) Run the app
3) Run the automated tests
### Who can test

Anyone except @weapdiv-david

### CODE REVIEW

**Comment by warrenbailey** [comment]
I thought it was simple.

However the Error page one isn't work, it gets override in the errors/error.html template 👎

File: tests/integration/mci/test_happy_path.py [hunk 1]
```diff
@@ -72,4 +75,5 @@ def happy_path(self,form_type_id):
 
         # We are on the thank you page
         content = resp.get_data(True)
+        self.assertRegexpMatches(content, '<title>Thank You</title>')
         self.assertRegexpMatches(content, '>Successfully Received<')
```
Comment (warrenbailey) [inline tests/integration/mci/test_happy_path.py:79]: I think it's worth adding a test for the error page

Comment (weapdiv-david) [inline tests/integration/mci/test_happy_path.py:79]: Agreed, updated in next commits

**Comment by warrenbailey** [comment]
LGTM 👍

**Pull request merged**